### ISSUE TITLE
CSSLint raising error for justify-content

### ISSUE THREAD

**Issue by KVGarg** [CSSLint/csslint#757]
I'm using property `space-evenly` with `justify-content`, and running CSSLint over my file is raising the follwoing error:
```bash
resources/css/style.css
[  35] ··justify-content:·space-evenly;
**** CSSLintBear [Section: css | Severity: NORMAL] ****
!    ! Expected (flex-start | flex-end | center | space-between | space-around) but found 'space-evenly'. (known-properties)
```

Also, same issue for `word-break`
```bash
resources/css/style.css
[ 177] ··word-break:·break-word;
**** CSSLintBear [Section: css | Severity: NORMAL] ****
!    ! Expected (normal | keep-all | break-all) but found 'break-word'. (known-properties)
```
This shouldn't happen!

**Comment by KVGarg** [comment]
@XhmikosR Can we get this fix possible, in next release?
